frontend/src/shared/helpers.tsx

```diff
@@ -576,40 +576,44 @@ export const CohortSamplesDetailsColumns: ColDef[] = [
     headerName: "CMO Sample Name",
   },
   {
+    field: "deliveryDate",
+    headerName: "Delivery Date",
+  },
+  {
+    field: "bamCompleteDate",
     headerName: "BAM Complete Date",
-    valueGetter: ({ data }) => data.bamComplete?.date,
   },
   {
+    field: "bamCompleteStatus",
     headerName: "BAM Complete Status",
-    valueGetter: ({ data }) => data.bamComplete?.status,
   },
   {
+    field: "mafCompleteDate",
     headerName: "MAF Complete Date",
-    valueGetter: ({ data }) => data.mafComplete?.date,
   },
   {
+    field: "mafCompleteNormalPrimaryId",
     headerName: "MAF Complete Normal Primary ID",
-    valueGetter: ({ data }) => data.mafComplete?.normalPrimaryId,
   },
   {
+    field: "mafCompleteStatus",
     headerName: "MAF Complete Status",
-    valueGetter: ({ data }) => data.mafComplete?.status,
   },
   {
+    field: "qcCompleteDate",
     headerName: "QC Complete Date",
-    valueGetter: ({ data }) => data.qcComplete?.date,
   },
   {
+    field: "qcCompleteResult",
     headerName: "QC Complete Result",
-    valueGetter: ({ data }) => data.qcComplete?.result,
   },
   {
+    field: "qcCompleteReason",
     headerName: "QC Complete Reason",
-    valueGetter: ({ data }) => data.qcComplete?.reason,
   },
   {
+    field: "qcCompleteStatus",
     headerName: "QC Complete Status",
-    valueGetter: ({ data }) => data.qcComplete?.status,
   },
```

Comment:
These changes are part of an object that defines the columns of the cohort samples popup. Instead of navigating inside the `data` object and grabbing the appropriate value for each field, I'm making these fields available at the top level. 

This new way of getting data...
```
  {
    field: "qcCompleteReason",
    headerName: "QC Complete Reason",
  }
```

...is equivalent to doing the following:
```
  {
    headerName: "QC Complete Reason",
    valueGetter: ({ data }) => data.qcCompleteReason
  },
```

The reason is to match the same pattern as `SampleDetailsColumns`, which defines the columns of the samples popup in other pages. 

### Example

Instead of nesting these fields in data like this:
```
const data = {
    qcComplete: {
        status: someValue,
        ...
    },
    ....
}
```

I'm making them available at the top level, like this:
```
const data = {
    qcCompleteStatus: someValue,
    ...
}
```